### PR TITLE
Add support for placeholder in commonconfig

### PR DESCRIPTION
**Pull request by msamsel**
It's necessary to make changes here. Umberto is not aware of this file and it seems to be more consistent to keep this change here.

### CODE REVIEW

File: scripts/tools/updateconfigfile.js [hunk 1]
```diff
@@ -3,15 +3,18 @@
...
 
+'use strict';
+
 const fs = require( 'fs-extra' );
+const path = require( 'path' );
 
 module.exports = ( { configFileSrc, configFileDst } ) => new Promise( ( resolve, reject ) => {
-	Promise.all( [ fs.readJson( configFileSrc ), fs.readFile( configFileDst ).then( data => data.toString() ) ] )
-		.then( ( [ srcConfig, dstFile ] ) => {
+	Promise.all( [ fs.readJson( configFileSrc ), fs.readFile( configFileDst ).then( data => data.toString() ), fs.readJson( path.join( process.cwd(), 'umberto.json' ) ) ] )
```
Comment (mlewand) [inline scripts/tools/updateconfigfile.js]: Please split it into multiline already.

**Comment by mlewand** [comment]
@msamsel please rebase it onto the latest `master`.

**Comment by msamsel** [comment]
rebased

**Comment by f1ames** [comment]
Rebased and retargeted on `stable` branch.

**Pull request merged**